docs/src/user_docs/assembly/u32_operations.md

```diff
@@ -14,6 +14,8 @@ For instructions where one or more operands can be provided as immediate paramet
 | u32test       | [a, ...]    | [b, a, ...]   | $b \leftarrow \begin{cases} 1, & \text{if}\ a < 2^{32} \\ 0, & \text{otherwise}\ \end{cases}$ |
 | u32testw      | [A, ...]    | [b, A, ...]   | $b \leftarrow \begin{cases} 1, & \text{if}\ \forall\ i \in \{0, 1, 2, 3\}\ a_i < 2^{32} \\ 0, & \text{otherwise}\ \end{cases}$ |
 | u32assert     | [a, ...]    | [a, ...]      | Fails if $a \ge 2^{32}$ |
+| u32assert.1   | [a, ...]    | [a, ...]      | Fails if $a \ge 2^{32}$ |
```

Comment:
I'd probably combine `u32assert` and `u32assert.1` into a single row, and would just mention in the comments that `1` can be omitted.

Reply:
Done